frontend/src/javascript/utils/axiosUtils.js

```diff
@@ -3,7 +3,7 @@ import toast, { getEndpointRelatedToast } from "@/javascript/utils/toastUtils";
 
 export function sendRequest(url, data = {}, method) {
     if (!validateMethod(method)) return;
-
+    axios.defaults.withCredentials = true;
     if (!data) data = {};
     data.withCredentials = true;
```

Comment:
Ta linia jest teraz już niepotrzebna z tym defaultem. To poprzednie rozwiązanie było takim obejściem żeby nie wysyłać credentials do wszystkich zapytań, ale zdecydowanie nie najlepszym. Myślę, że na razie to nie jest problem, bo nie wysyłamy zapytań poza nasz ekosystem.